monkey/monkey_island/cc/ui/src/services/configuration/propagation/scanTarget.js

```diff
@@ -18,38 +16,17 @@ const SCAN_TARGET_CONFIGURATION_SCHEMA = {
       'default': [],
       'description': 'List of IPs that the monkey will not scan.'
     },
-    'inaccessible_subnets': {
-      'title': 'Network segmentation testing',
-      'type': 'array',
-      'uniqueItems': true,
-      'items': {
-        'type': 'string',
-        'format': 'ip-range'
-      },
-      'default': [],
-      'description': 'Test for network segmentation by providing a list of network segments that should NOT be accessible to each other.\n\n ' +
-        'For example, if you configured the following three segments: ' +
-        '"10.0.0.0/24", "11.0.0.2/32" and "12.2.3.0/24",' +
-        'a Monkey running on 10.0.0.5 will try to access machines in ' +
-        'the following subnets: ' +
-        '11.0.0.2/32, 12.2.3.0/24. An alert on successful cross-segment connections ' +
-        'will be shown in the reports. \n\n' +
-        'Network segments can be IPs, subnets or hosts. Examples:\n' +
-        '\tDefine a single-IP segment: "192.168.0.1"\n' +
-        '\tDefine a segment using a network range: ' +
-        '"192.168.0.5-192.168.0.20"\n' +
-        '\tDefine a segment using an subnet IP mask: "192.168.0.5/24"\n' +
-        '\tDefine a single-host segment: "printer.example"'
+    'info_box_scan_my_networks': {
+      'info': 'If "Scan Agent\'s networks" is enabled, the Agent will go over all network ' +
+        'interfaces and will scan their networks, ' +
+        'in addition to the IPs that are configured manually in the "Scan target list". ' +
```

Comment:
I somehow feel we lose readability by being super specific about what's happening. Maybe "Agent will scan all networks it's a part of"? I'm not sure, maybe @mssalvatore could help

Reply:
We talked on team time about creating a warning box instead of info and shortening the language.